models/balancer/balancer_ethereum_balances.sql

```diff
@@ -0,0 +1,66 @@
+    WITH pools AS (
+        SELECT pool as pools
+        FROM {{ source('balancer_ethereum', 'BFactory_evt_LOG_NEW_POOL') }}
+
+    ),
+    
+    joins AS (
+        SELECT p.pools as pool, date_trunc('day', e.evt_block_time) AS day, e.contract_address AS token, SUM(value) AS amount
+        FROM {{ source('erc20_ethereum', 'evt_transfer') }} e 
+        INNER JOIN pools p ON e.`to` = p.pools
+        GROUP BY 1, 2, 3
+
+        UNION ALL
+
+        SELECT e.`to` as pool, date_trunc('day', e.evt_block_time) AS day, e.contract_address AS token, SUM(value) AS amount
+        FROM {{ source('erc20_ethereum', 'evt_transfer') }} e
+        WHERE e.`to` = '0xBA12222222228d8Ba445958a75a0704d566BF2C8'
```

Comment:
Here and below as well, you need a lower-case address. Simple way to convert:
https://dune.com/queries/1274287